packages/libretro/retroarch/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="retroarch"
-PKG_VERSION="3a981e0"
+PKG_VERSION="b3d3cbd"
```

Comment:
I think we should go with https://github.com/libretro/RetroArch/commit/6fc6bfbb243de4da05a86d1edc3950815a964f1e, which is 1.7.1 release